gpu-operator/gpu-operator-rdma.rst

```diff
@@ -30,10 +30,9 @@ To support GPUDirect RDMA, userspace CUDA APIs are required.
 The kernel mode support is provided by one of two approaches: DMA-BUF from the Linux kernel or the legacy ``nvidia-peermem`` kernel module.
 NVIDIA recommends using the DMA-BUF rather than using the ``nvidia-peermem`` kernel module from the GPU Driver.
 
-Starting with v23.9.1 of the Operator, the Operator uses GDS driver version 2.17.5 or newer.
+The Operator uses GDS driver version 2.17.5 or newer.
 This version and higher is only supported with the NVIDIA Open GPU Kernel module driver.
-The sample commands for installing the Operator include the ``--set useOpenKernelModules=true``
-command-line argument for Helm.
+Newer driver versions will use an open kernel module by default, however to make sure you are using an open model, you can include the ``--set driver.kernelModuleType=open`` command-line arugment in your Operator install command.
```

Comment:
IMO We should be more explicit here.  Let's mention that the supported driver versions of the gpu-operator will support the new `auto` mode, which will choose the recommended driver kernel module type depending on the driver branch and the GPU device

Reply:
i updated the wording here, can you take another look?

Reply:
Yes, looks great, thank you! Added one more comment